servers/rendering_server.cpp

```diff
@@ -2474,6 +2474,7 @@ void RenderingServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("light_directional_set_sky_mode", "light", "mode"), &RenderingServer::light_directional_set_sky_mode);
 
 	ClassDB::bind_method(D_METHOD("light_projectors_set_filter", "filter"), &RenderingServer::light_projectors_set_filter);
+	ClassDB::bind_method(D_METHOD("lightmaps_set_bicubic_filter", "filter"), &RenderingServer::lightmaps_set_bicubic_filter);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("lightmaps_set_bicubic_filter", "enable"), &RenderingServer::lightmaps_set_bicubic_filter);
```
To match the docs. Might also be worth changing the C++ methods argument from `p_use` to `p_enable` for consistency, but it's not required.